Tests/MongoSwiftTests/DocumentTests.swift

```diff
@@ -680,4 +682,122 @@ final class DocumentTests: MongoSwiftTestCase {
         expect(doc["autoclosure test", default: "\(stringVal) and \(floatVal)" + stringVal])
             .to(bsonEqual("\(stringVal) and \(floatVal)" + stringVal))
     }
+
+    func testUUIDCodingStrategies() throws {
+        // TODO: add encoding strategy tests SWIFT-282
+
+        // swiftlint:disable nesting
```

Comment:
should we disable this linter rule? I don't really see the point

Reply:
I'll do so in my linting PR. for now we can leave this, if this gets merged first then I'll deal with it in my PR.